packages/components/src/notice/style.scss

```diff
@@ -34,6 +34,9 @@
 	&.is-link {
 		margin-left: 4px;
 	}
+	&.is-default {
+		vertical-align: unset;
```

Comment:
`unset` is not supported by IE11: maybe try `initial` (baseline) or remove it all together?

Reply:
Hi @afercia thank you for looking into this PR and suggesting alternatives. I used your fist suggestion and now vertical-align is set to 'initial'. Removing the rule all together would make buttons inside notices not aligned with the notice content.